python/hail/methods/statgen.py

```diff
@@ -1604,10 +1606,23 @@ def split_multi_hts(ds, keep_star=False, left_aligned=False) -> MatrixTable:
 
     """
 
+    row_fields = set(ds.row)
     entry_fields = set(ds.entry)
     split = split_multi(ds, keep_star=keep_star, left_aligned=left_aligned)
+    update_rows_expression = {}
     update_entries_expression = {}
 
+    if vep_root in row_fields:
```

Comment:
will this work for all VEPs or just human VEP? with the recent changes, we should try to be flexible.

Also I think this is broken -- a_index is a field, not a method

Reply:
yep, good catch. some old code copied from back when it was

Reply:
and i _believe_ all VEPs add these 4 fields, but i could be run. are you guys testing mouse VEP? or has anyone over there run it and can show me what the results look like?